simulator/src/resim/cmd_show_ledger.rs

```diff
@@ -40,7 +40,7 @@ impl ShowLedger {
         )
         .map_err(Error::IOError)?;
 
-        let instant = Self::get_current_time(out, TimePrecision::Minute)?;
+        let instant = Self::get_current_time(out, TimePrecisionV1::Minute)?;
```

Comment:
Hrm, was thinking of updating resim at a later PR so that users don't build against interfaces which don't exist on ledger yet but I think that might be fine so will implement your updates

Reply:
I think that if we plan on releasing a new version of Scrypto on the day of the protocol update then it would be fine if resim was updated.

Reply:
Running into some issues right now with getting this implemented. As discussed with @0xOmarA , he will create a follow-up PR fixing resim.